scaaml/capture/aes/acqktp.py

```diff
@@ -215,4 +215,4 @@ def new_pair(self):
     newPair = camel_case_deprecated(new_pair)
 
     def __str__(self):
-        return f'{self._name} ({self._dataset}, {self._pt_per_key}, {self._repeat}, {self._nbkeys})'
+        return f"{self._name} ({self._dataset}, {self._pt_per_key}, {self._repeat}, {self._nbkeys})"
```

Comment:
I'm surprised this line isn't flagged as too long. At first glance it looks longer than 80 chars but I can be wrong